frontend/src/components/ControlPlaneHighAvailability/ControlPlaneHighAvailabilityTag.vue

```diff
@@ -0,0 +1,110 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <g-popper
+    v-if="shootControlPlaneHighAvailabilityFailureTolerance"
+    title="Control Plane High Availability"
+    :popper-key="`cp_ha_tag_${shootControlPlaneHighAvailabilityFailureTolerance}`"
+    :toolbar-color="color"
+  >
+    <template v-slot:popperRef>
+      <v-chip
+        outlined
+        :small="!xSmall"
+        :x-small="xSmall"
+        :color="color"
+        class="cursor-pointer ml-1"
+      >
+        {{shootControlPlaneHighAvailabilityFailureTolerance}}
+      </v-chip>
+    </template>
+    <v-list>
```

Comment:
```suggestion
    <v-list  class="ma-n4 text-left" style="max-width: 600px;">
```
We don't need the `::-v-deep` here I think. I cannot find a message class in GPopper.